src/remote-config/remote-config-api.ts

```diff
@@ -105,6 +105,11 @@ export interface OneOfCondition {
    * Makes this condition a percent condition.
    */
   percent?: PercentCondition;
+
+  /**
+   * Makes this a custom signal condition.
+   */
```

Comment:
If we haven't done so already, could we get a TW review on the docstrings here (and below), please?

Reply:
added @jenh!